Sming/Services/Yeelight/YeelightBulb.h

```diff
@@ -0,0 +1,51 @@
+/****
+ * Sming Framework Project - Open Source framework for high efficiency native ESP8266 development.
+ * Created 2015 by Skurydin Alexey
+ * http://github.com/anakod/Sming
+ * All files of the Sming Core are provided under the LGPL v3 license.
+ ****/
+
+#include "../Wiring/WiringFrameworkDependencies.h"
+#include "../Wiring/WHashMap.h"
+#include "../Wiring/WVector.h"
+#include "../Wiring/WString.h"
+#include "IPAddress.h"
+class TcpClient;
+
+#ifndef SERVICES_YEELIGHTBULB_H_
+#define SERVICES_YEELIGHTBULB_H_
+
+class YeelightBulb
+{
+public:
+	YeelightBulb(IPAddress addr);
```

Comment:
@anakod Can you please add documentation comments? Take a look at [that document](https://github.com/SmingHub/Sming/wiki/Documenting-the-API) if you need guiding lines.